cms/templates/settings.html

```diff
@@ -94,10 +96,10 @@ <h3 class="title">${_("Course Summary Page")} <span class="tip">${_("(for studen
               <li class="action-item">
                 <%
                   email_subject = urllib.quote(_("Enroll in {course_display_name}").format(
-                      course_display_name = context_course.display_name_with_default_escaped
+                      course_display_name = context_course.display_name_with_default
                   ).encode("utf-8"))
                   email_body = urllib.quote(_('The course "{course_display_name}", provided by {platform_name}, is open for enrollment. Please navigate to this course at {link_for_about_page} to enroll.').format(
-                      course_display_name = context_course.display_name_with_default_escaped,
+                      course_display_name = context_course.display_name_with_default,
                       platform_name = settings.PLATFORM_NAME,
                       link_for_about_page = link_for_about_page
```

Comment:
This is a tricky one that the linter would miss.  I imagine that `link_for_about_page` should be wrapped with `HTML()`, and you will thus need to wrap the initial string with `Text()`.  Is this `email_body` used for HTML only?  If it is used for HTML and plain text emails, we may need `email_body_html` and `email_body_plain_text`.
Also, why are we using `urllib.quote`?  Are these being used in a URL?